lib/src/style/Theme.cpp

```diff
@@ -175,8 +207,8 @@ std::optional<Theme> Theme::fromJsonDoc(const QJsonDocument& jsonDoc) {
 
 void Theme::initializeFonts() {
   // Fonts.
-  const auto defaultFont = QFont(QStringLiteral("Inter"));
-  const auto fixedFont = QFont(QStringLiteral("Roboto Mono"));
+  const auto defaultFont = useSystemFont ? QFont() : QFont(QStringLiteral("Inter"));
+  const auto fixedFont = useSystemFont ? QFont() : QFont(QStringLiteral("Roboto Mono"));
```

Comment:
You can use this instead:

- Default font : `QFontDatabase::systemFont(QFontDatabase::GeneralFont)`.
- Fixed font : `QFontDatabase::systemFont(QFontDatabase::FixedFont)`.

Reply:
Also, I've just seen that`QFontDatabase::TitleFont` exists!

Since Qlementine has fonts for headings/titles, it would be cool to add Inter-Display, Inter's tighter variant for headings.

Maybe in another PR? https://github.com/oclero/qlementine/issues/89

Reply:
I could add it to this PR...shouldn't be too hard